github/github_test.go

```diff
@@ -779,3 +720,75 @@ func TestReviewPriorPRs(t *testing.T) {
 
 	assert.NoError(t, ReviewPriorPRs(logger, mockDB, &user))
 }
+
+func getSignedSignatureVerification() *github.SignatureVerification {
+	return &github.SignatureVerification{
+		Verified:  github.Bool(true),
+		Reason:    github.String("valid"),
+		Signature: github.String("some-signature"),
+		Payload:   github.String("some-payload"),
+	}
+}
+
+func getUnsignedSignatureVerification() *github.SignatureVerification {
+	return &github.SignatureVerification{
+		Verified:  github.Bool(false),
+		Reason:    github.String("unsigned"),
+		Signature: nil,
+		Payload:   nil,
+	}
+}
+
+func getMockRepositoryCommits(mockAuthorLogins []string, signed bool) []*github.RepositoryCommit {
+	mockRepositoryCommits := make([]*github.RepositoryCommit, 0)
+
+	for _, author := range mockAuthorLogins {
+		email := fmt.Sprintf("%s@somewhere.tld", author)
+		var signatureVerification *github.SignatureVerification = getSignedSignatureVerification()
```

Comment:
minor:  type can be removed

```suggestion
		var signatureVerification = getSignedSignatureVerification()
```